### DISCUSSION TITLE
How could I version my pod into git?

### DISCUSSION THREAD

**Discussion by rolivegab** [containers/podman#12015] [Q&A] [answered]
With docker, we have docker-compose.yaml,
and with kubernetes, we can have .yaml files on git.

but with podman, I think we don't have a oficial solution, that make possible have my infrastructure on git, and deploy them with some command out of the box.

**Comment by rhatdan** [comment, marked as answer]
podman play kube can use the kubernetes.yaml file.

**Reply by rolivegab** [reply]
Thank you, I'll see about it!

**Comment by rhatdan** [comment]
Podman can also use the docker-compose as well

**Comment by vrothberg** [comment]
The following blog could be a good starting point, covering both K8s yaml and compose: https://www.redhat.com/sysadmin/compose-podman-pods
